src/napari/layers/points/points.py

```diff
@@ -2498,6 +2498,10 @@ def update_selected_view(self):
             )[2]
         )
 
+        # WARNING This wil lbe removed in future
+        with self.layer.events.highlight.blocker():
+            self.layer._set_highlight(force=True)
+
```

Comment:
why is this needed? Add it to the comment, so we also know *when* we can remove it in the future.

Reply:
You just pointed out that we are mising ths a comment before
 https://github.com/napari/napari/pull/8254/files/c62a0a551b326b87e23bd7ec801499a1d555d781#diff-feef49e7b65852afee1679897aec2ec4bcf8d086ccef004fdfe410c229a73c21